assets/components/yourDetails/yourDetails.jsx

```diff
@@ -24,6 +24,7 @@ export default function YourDetails(props: PropTypes) {
     <div className="component-your-details">
       <PageSection heading="Your details" headingChildren={<Signout />}>
         <DisplayName />
+        <p className="component-your-details-text">All fields are required.</p>
```

Comment:
This is a [BEM element](http://getbem.com/naming/), so `component-your-details__text`.

This copy will also appear on the one-off checkout, just checking that's what you intended?

Reply:
Ah good spot. It wasn't my intention, but all fields are required for one-off too, so it makes sense in the context of the page.